src/app/move-popup/Sockets.tsx

```diff
@@ -71,13 +74,26 @@ export default class Sockets extends React.Component<Props, State> {
                 {!hideMods && (
                   <div className="item-socket-category-name">
                     <div>{category.category.displayProperties.name}</div>
-                    {anyBestRatedUnselected(category) && (
-                      <div className="best-rated-key">
-                        <div className="tip-text">
-                          <BestRatedIcon /> {t('DtrReview.BestRatedKey')}
+                    {(!curationEnabled ||
+                      !inventoryCuratedRoll ||
+                      !inventoryCuratedRoll.isCuratedRoll) &&
+                      anyBestRatedUnselected(category) && (
+                        <div className="best-rated-key">
+                          <div className="tip-text">
+                            <BestRatedIcon curationEnabled={false} /> {t('DtrReview.BestRatedKey')}
+                          </div>
                         </div>
-                      </div>
-                    )}
+                      )}
+                    {curationEnabled &&
+                      inventoryCuratedRoll &&
+                      inventoryCuratedRoll.isCuratedRoll &&
+                      anyCuratedRolls(category, inventoryCuratedRoll) && (
+                        <div className="best-rated-key">
+                          <div className="tip-text">
+                            <BestRatedIcon curationEnabled={true} /> {t('CuratedRoll.BestRatedKey')}
```

Comment:
(prayer hands emoji)

Reply:
This isn't working out quite right - it keeps rendering `[object Object] = curator-suggested perk` rather than the icon.

I tried a few variations on it, but it's not super working.

Reply:
no worries it was worth a try

Reply:
I'd like to get it sorted out, but it's beta for now.